File: sct_source/SourceCode/FE_SoCongThuong/Angular/src/app/modules/app-pages/industrial-cluster-info-management-page/_models/cate-industrial-cluster.model.ts
import { BaseModel } from '../../../../_metronic/shared/crud-table/models/base.model';

export interface CateIndustrialClusterModel extends BaseModel {
  cateIndustrialClustersId : string;
  industrialClustersName: string; //Tên cụm công nghiệp
  originalArea: number | null; //Diện tích thành lập
  establishCode: string; //Quyết định thành lập
  expandedArea: number | null; //Diện tích mở rộng
  decisionExpandCode: string; //Quyết định mở rộng
  detailedArea: number | null; //Diện tích QH chi tiết
  approvalDecision: string; //Quyết định phê duyệt QHCT
  industrialArea: number | null; //Diện tích đất công nghiệp
  rentedArea: number | null; //Diện tích đất cho thuê
  occupancy: number | null; //Tỉ lệ lấp đấy
  note: string; //Ghi chú
  district: string;
}

File: sct_source/SourceCode/API_SoCongThuong/Models/CateIndustrialClusterModel.cs
using System;
using System.Collections.Generic;

namespace API_SoCongThuong.Models
{
    public partial class CateIndustrialClusterModel
    {
        public Guid CateIndustrialClustersId { get; set; }
        public string IndustrialClustersName { get; set; } = null!;
        public int OriginalArea { get; set; }
        /// <summary>
        /// Quyết định thành lập
        /// </summary>
        public string EstablishCode { get; set; } = null!;
        /// <summary>
        /// Diện tích mở rộng
        /// </summary>
        public int ExpandedArea { get; set; }
        /// <summary>
        /// Quyết định mở rộng
        /// </summary>
        public string DecisionExpandCode { get; set; } = null!;
        /// <summary>
        /// Diện tích chi tiết
        /// </summary>
        public int DetailedArea { get; set; }
        /// <summary>
        /// quyết định phê duyệt
        /// </summary>
        public string ApprovalDecision { get; set; } = null!;
        /// <summary>
        /// Diện tích đất công nghiệp
        /// </summary>
        public int IndustrialArea { get; set; }
        /// <summary>
        /// Diện tích đã cho thuê
        /// </summary>
        public int RentedArea { get; set; }
        /// <summary>
        /// Tỷ lệ lấp đầy
        /// </summary>
        public int Occupancy { get; set; }
        public string? Note { get; set; }
        /// <summary>
        /// 1: Hoạt động 0: Không hoạt động
        /// </summary>
        public bool? IsAction { get; set; }
        /// <summary>
        /// 1: Đã xóa; 0: Chưa xóa
        /// </summary>
        public bool IsDel { get; set; }
        /// <summary>
        /// Thời gian tạo
        /// </summary>
        public DateTime CreateTime { get; set; }
        public string CreateTimeDisplay { get; set; } = "";
        /// <summary>
        /// Người tạo
        /// </summary>
        public Guid CreateUserId { get; set; }
        public string CreateName { get; set; } = "";
        public DateTime? UpdateTime { get; set; }
        public Guid? UpdateUserId { get; set; }
        public Guid? District { get; set; } = Guid.Empty;
    }
}
